homeassistant/components/snmp/switch.py

```diff
@@ -190,14 +190,25 @@ def __init__(
 
     async def async_turn_on(self, **kwargs):
         """Turn on the switch."""
-        await self._set(self._command_payload_on)
+        from pyasn1.type.univ import Integer
```

Comment:
Move the imports to the top of the file.

Reply:
Thanks for the comment; I moved it up.
It's indeed cleaner; I had just restored that from how it was in 0.100.3 without thinking about that 🙈